app/survey_config/business_config.py

```diff
@@ -134,10 +132,7 @@ def _stripped_base_url(self) -> str:
 
 @dataclass
 class NorthernIrelandBusinessSurveyConfig(BusinessSurveyConfig):
-
-    page_header_logo: str = "ni-finance-logo"
-    page_header_logo_alt: str = lazy_gettext(
-        "Northern Ireland Department of Finance logo"
-    )
-    mobile_logo: str = "ni-finance-logo-mobile"
-    custom_header_logo: bool = True
+    with open("./img/ni-finance-logo.svg", encoding="UTF-8") as f:
```

Comment:
We should make use of the `read_file` method in settings.py in place of these `with open` calls